src/main/java/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,45 @@
+package controller;
+
+import dao.BetDao;
+import dao.UserDao;
+import java.util.Scanner;
+import lib.Inject;
+import model.Bet;
+import model.User;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    UserDao userDao;
+
+    public void handle() {
+        String splitter = " ";
```

Comment:
this can be removed
or made a constant
or moved closer to its usage (inside `try` block)